### DISCUSSION TITLE
Lesson 10 Aave Website Metamask Question

### DISCUSSION THREAD

**Discussion by NewCoderrrrr** [smartcontractkit/full-blockchain-solidity-course-py#1302] [Q&A] [answered]
On the website: https://classic.aave.com/#/deposit (I had to use the classic version of this website because the deposit tab is not on the newer version), it will not let me link my metamask wallet unless I am connected to the Ethereum Mainnet. It even says on their website: https://docs.aave.com/faq/troubleshooting , "Make sure to select the correct network (Ethereum Mainnet). Normally on the wallet provider you can switch the network in the settings option".  Did they remove Kovan functionality or is there a way to fix this?

**Comment by cromewar** [comment, marked as answer]
Hello @NewCoderrrrr 
The reason is you must use the testnet version for aave.
https://v3-test.aave.com/#/markets
https://staging.aave.com/
I hope this might help

**Reply by NewCoderrrrr** [reply]
Thanks
